contrib/format-hdf5/src/main/java/org/apache/drill/exec/store/hdf5/HDF5BatchReader.java

```diff
@@ -946,7 +988,11 @@ private void writeAttributes(TupleWriter rowWriter, HDF5DrillMetadata record) {
           writeLongColumn(mapWriter, key, (Long) attrib.getValue());
           break;
         case INT:
-          writeIntColumn(mapWriter, key, (Integer) attrib.getValue());
+          try {
+            writeIntColumn(mapWriter, key, (Integer) attrib.getValue());
+          } catch (Exception e) {
+            System.out.println(key +  " " + attrib.toString());
```

Comment:
Please remove and do proper error handling.

Reply:
Crap... sorry I missed that.  Removed.